test/unit/spec/contract_spec.ts

```diff
@@ -1,12 +1,14 @@
-import { xdr, Address, contract, Keypair } from "../../../lib";
+import { xdr, Address, contract, Keypair, hash } from "../../../lib";
 import { JSONSchemaFaker } from "json-schema-faker";
 
 import spec from "../spec.json";
+import spec_contstructor from "../spec_constructor.json";
```

Comment:
lol `contstructor` weird typo?

Reply:
obviously a portmanteau of "contract" and "constructor", don't see what's so weird about it 😐

Reply:
(resolved now)